src/Components/MyListsHandler.cs

```diff
@@ -288,106 +315,192 @@ private void ResetSearchAndDisplayIngredients(TextBox searchBox, StackPanel ingr
+                FontSize = ingTextFont - 4,
+            };
 
-            // Create move button with text
-            Button moveButton = new Button
+            // Ingredient amount, unit text 
+            TextBlock ingredientAmountText = new TextBlock
             {
-                Content = "Move",
-                FontSize = 16,
-                Margin = new Thickness(5, 0, 0, 0),
-                Background = Brushes.White
+                Text = $"{ingredient.GetAmount()} {ingredient.GetUnit()}",
+                Foreground = ingTextCol,
+                VerticalAlignment = VerticalAlignment.Center,
+                FontSize = ingTextFont - 4,
             };
-            moveButton.Click += async (s, e) => await ShowMoveIngredientPopup(userList.GetListName(), ingredient);
 
-            // Add move button to the panel 
-            buttonPanel.Children.Add(moveButton);
+            DockPanel.SetDock(ingredientNameText, Dock.Top);
+            ingredientRow.Children.Add(ingredientNameText);
+            DockPanel.SetDock(ingredientTypeText, Dock.Top);
+            ingredientRow.Children.Add(ingredientTypeText);
+            DockPanel.SetDock(ingredientAmountText, Dock.Top);
+            ingredientRow.Children.Add(ingredientAmountText);
 
-            // Delete button (Trash icon)
+            // Panel to hold delete, edit, and move buttons
+            StackPanel buttonPanel = new StackPanel
+            {
+                Orientation = Orientation.Horizontal,
+                HorizontalAlignment = HorizontalAlignment.Right
+            };
+
+            // Move button
+            Button moveButton = new Button
+            {
+                Content = "\u21C4",
+                FontSize = buttonFont + 6,
+                FontWeight = FontWeights.Bold,
+                Margin = new Thickness(20, 10, 5, 5),
+                Background = Brushes.Transparent,
+                BorderBrush = Brushes.Transparent,
+                Foreground = buttonCol,
+                Cursor = Cursors.Hand,
+                ToolTip = "Move"
+            };
+            moveButton.Style = (Style)Application.Current.Resources["NoHighlightButton"];
+            moveButton.Click += async (s, e) => await ShowMoveIngredientPopup(userList, ingredient, ingPanel);
+
+            // Delete button
             Button deleteButton = new Button
             {
                 Content = "\uD83D\uDDD1", // Trash can icon
-                Width = 30,
-                Height = 30,
-                Background = Brushes.White,
-                Margin = new Thickness(10, 0, 0, 0)
+                FontSize = buttonFont,
+                FontWeight = FontWeights.Bold,
+                Background = Brushes.Transparent,
+                BorderBrush = Brushes.Transparent,
+                Foreground = buttonCol,
+                Margin = new Thickness(20, 10, 5, 5),
+                Cursor = Cursors.Hand,
+                ToolTip = "Delete"
```

Comment:
Delete button cutoff on the screen. Make it relative. This probably happened due to long ingredient text name.
![image](https://github.com/user-attachments/assets/8ba8b69b-cc5b-4848-9977-7e8cf109ec55)

Also make the text on delete button red to indicate negative action.